jekyll/_cci2/orb-concepts.md

```diff
@@ -7,203 +7,171 @@ categories: [getting-started]
-
-### Orb Structure
-
-Orbs consist of the following elements:
-
-- Commands
-- Jobs
-- Executors
-
-#### Commands
+```yaml
+version: 2.1
 
-Commands are reusable sets of steps that you can invoke with specific parameters within an existing job. For example, if you want to invoke the command `sayhello`, you would pass the parameter to as follows:
+orbs:
+  aws-s3: circleci/aws-s3@x.y.z
 
-```
-version: 2.1
 jobs:
-  myjob:
+  build:
     docker:
-      - image: "circleci/node:9.6.1"
+      - image: 'cimg/python:3.6'
     steps:
-      - myorb/sayhello:
-          to: "Lev"
-```
-
-#### Jobs
-
-Jobs are comprised of two parts: a set of steps, and the environment they should be executed within. Jobs are defined in your build configuration or in an orb and enable you to define a job name in a map under the jobs key in a configuration, or in an external orb’s configuration.
-
-You must invoke jobs in the workflow stanza of config.yml file, making sure to pass any necessary parameters as subkeys to the job.
+      - checkout
+      - run: mkdir bucket && echo "lorem ipsum" > bucket/build_asset.txt
+      - aws-s3/copy:
+          from: bucket/build_asset.txt
+          to: 's3://my-s3-bucket-name'
 
-#### Executors
-
-Executors define the environment in which the steps of a job will be run. When you declare a job in CircleCI configuration, you define the type of environment (e.g. docker, machine, macos, etc.) to run in, in addition to any other parameters of that environment, such as:
-
-- environment variables to populate
-- which shell to use
-- what size resource_class to use
-
-When you declare an executor in a configuration outside of jobs, you can use these declarations for all jobs in the scope of that declaration, enabling you to reuse a single executor definition across multiple jobs.
+... # workflows , other jobs etc.
+```
 
-An executor definition has the following keys available (some of which are also available when using the job declaration):
+### Executors
 
-- docker, machine, or macos
-- environment
-- working_directory
-- shell
-- resource_class
+Executors define the environment in which commands are run. CircleCI provides multiple [execution platforms]({{site.baseurl}}/2.0/configuration-reference/#docker--machine--macos--windows-executor) including:
+  - Docker
+  - macOS
+  - Windows
+  - Machine (Linux VM)
 
-The example below shows a simple example of using an executor:
+In the [Node orb](https://circleci.com/orbs/registry/orb/circleci/node), for example, a parameterized Docker-based executor is provided, through which users can set the Docker tag. This provides a simple way to test applications against any version of Node.js when used with the Node orb's [test job](https://circleci.com/orbs/registry/orb/circleci/node#usage-run_matrix_testing).
 
+```yaml
+docker:
+  - image: '<org>/<image>:<<parameters.tag>>'
+parameters:
+  tag:
+    default: '13.11'
+    type: string
 ```
```

Comment:
This doesn't quite match with the previous example - should this be showing how to use an orb's executor, or how to define one?

Reply:
Ya I agree. Going to switch this more to an example of how. What I would like to communicate (but in a better way) is the purpose of executors, as many folks add them for no real reason in some cases. That language could be put elsewhere too, more with authoring.